headers/types/common/common.h

```diff
@@ -36,6 +53,47 @@ struct file_stream {
 };
 ASSERT_SIZE(struct file_stream, 72);
 
+// Struct used to load, store and unload WTE files
+struct wte_handle {
+    void* content; // Pointer to the heap-allocated WTE data. Only stored for freeing the data
+    struct wte_header* header;
+};
+ASSERT_SIZE(struct wte_handle, 8);
+
+// These arguments are almost directly passed to the TEXIMAGE_PARAM register, just rearranged
+// For more information see:
+// https://problemkaputt.de/gbatek.htm#ds3dtextureattributes
+struct wte_texture_params {
+    uint8_t texture_smult : 3;
+    uint8_t texture_tmult : 3;
+    uint8_t unused6 : 2;
+    enum texture_format format : 3;
+    bool flip_x : 1;
+    bool flip_y : 1;
+};
```

Comment:
Are the last 3 bits unused? Can we add a `uint8_t unused13 : 3` at the end for explicitness?